src/clients/storage/BlobStorageClient.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+let request = require('request');
+const apiUrlBase = process.env.FORTIS_CENTRAL_ASSETS_HOST || 'https://fortiscentral.blob.core.windows.net/';
+
+/**
+ * @param {string} siteType
+ * @returns {string}
+ */
+function buildTopicsSiteTypeUri(siteType) {
+  return `${apiUrlBase}settings/siteTypes/${siteType}/topics/defaultTopics.json`;
+}
+
+/**
+ * @param {string} siteType
+ * @returns {Promise.<Array<{keyword: string, lang_code: string, insertion_time: timestamp, translations: Map<string, string>}>>}
+*/
+function getTopicsBySiteType(siteType) {
```

Comment:
The `get` prefix usually indicates something cheap, like a field access. Here we're doing an expensive operation (network request) so it's more obvious to call it `fetch`. Here's a good [SO thread](https://stackoverflow.com/q/2141818/3817588) on the topic.